docs/src/developers_guide/contributing_graphics_tests.rst

```diff
@@ -2,72 +2,17 @@
+Adding or Updating Graphics Tests
+=================================
 
-Iris may be used to create various forms of graphical output; to ensure
-the output is consistent, there are automated tests to check against
-known acceptable graphical output.  See :ref:`developer_running_tests` for
-more information.
-
-At present graphical tests are used in the following areas of Iris:
-
-* Module ``iris.tests.test_plot``
-* Module ``iris.tests.test_quickplot``
-* :ref:`sphx_glr_generated_gallery` plots contained in
-  ``docs/gallery_tests``.
-
-
-Challenges
-==========
-
-Iris uses many dependencies that provide functionality, an example that
-applies here is matplotlib_.  For more information on the dependences, see
-:ref:`installing_iris`.  When there are updates to the matplotlib_ or a
-dependency of matplotlib, this may result in a change in the rendered graphical
-output.  This means that there may be no changes to Iris_, but due to an
-updated dependency any automated tests that compare a graphical output to a
-known acceptable output may fail.  The failure may also not be visually
-perceived as it may be a simple pixel shift.
-
-
-Testing Strategy
-================
-
-Our `Iris GitHub Actions`_ define multiple test runs that use
-different versions of Python to ensure Iris is working as expected.
-
-To make this manageable, the ``iris.tests.IrisTest_nometa.check_graphic`` test
-routine tests against multiple alternative **acceptable** results.  It does
-this using an image **hash** comparison technique which avoids storing
-reference images in the Iris repository itself.
-
-This consists of:
-
-* The ``iris.tests.IrisTest_nometa.check_graphic`` function uses a perceptual
-  **image hash** of the outputs (see https://github.com/JohannesBuchner/imagehash)
-  as the basis for checking test results.
-
-* The hashes of known **acceptable** results for each test are stored in a
-  lookup dictionary, saved to the repo file
-  ``lib/iris/tests/results/imagerepo.json``
-  (`link <https://github.com/SciTools/iris/blob/main/lib/iris/tests/results/imagerepo.json>`_) .
-
-* An actual reference image for each hash value is stored in a *separate*
-  public repository https://github.com/SciTools/test-iris-imagehash.
-
-* The reference images allow human-eye assessment of whether a new output is
-  judged to be close enough to the older ones, or not.
-
-* The utility script ``iris/tests/idiff.py`` automates checking, enabling the
-  developer to easily compare proposed new **acceptable** result images
-  against the existing accepted reference images, for each failing test.
+.. note::
 
-The acceptable images for each test can be viewed online. The :ref:`testing.imagehash_index` lists all the graphical tests in the test suite and
-shows the known acceptable result images for comparison.
+  If a large number of images tests are failing due to an update to the
+  libraries used for image hashing, follow the instructions on
+  :ref:`refresh-imagerepo`.
```

Comment:
Not sure where this link goes?

Reply:
Ah, my sneaky reminder to myself to write that has worked! I've now written this, thanks :)